### ISSUE TITLE
 Unable to select multiple entries in clarity grid using shift

### ISSUE THREAD

**Issue by poorvi12-vmware** [vmware/clarity#4450]
Hi,

There is no option to select multiple rows (anywhere from the row) in clarity grid using shift like we have in kendo grid. 

**How to reproduce**

1) Open clarity grid with multi select option.
2) Try to select multiple rows using shift and click in order to perform bulk actions.

**Version**

Clarity v2

Thanks

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
